exercises/practice/list-ops/test/list_ops_test.exs

```diff
@@ -76,6 +76,11 @@ defmodule ListOpsTest do
     test "huge list" do
       assert L.filter(Enum.to_list(1..1_000_000), &odd?/1) == Enum.map(1..500_000, &(&1 * 2 - 1))
     end
+
+    @tag :pending
+    test "non boolean results" do
```

Comment:
```suggestion
    test "truthy values filter the list" do
```

Reply:
Sure, sounds good. Done!

Reply:
Cool, this looks good, there is one check failing, can you run `mix format` from that exercise directory?